core/SymbolRef.h

```diff
@@ -397,6 +397,10 @@ class Symbols {
         return SymbolRef(nullptr, 83);
     }
 
+    static SymbolRef Lazy() {
```

Comment:
can you name this `Enumerator_Lazy` please?  The convention is to name it as close to the full path as makes sense

Reply:
Done 👍